### PR TITLE
Added removing /opt dir

### PR DESCRIPTION
**Pull request by JasminAnt**
Decommission command was leaving /opt dir behind

image

image
...
image

### CODE REVIEW

File: cmd/decomissionNode.go [hunk 1]
```diff
@@ -116,6 +116,8 @@ func decommissionNodeRun(cmd *cobra.Command, args []string) {
 		RunCommandWait("sudo pkill -9 `pidof etcd`")
 		RunCommandWait("sudo pkill -9 `pidof kube_proxy`")
 
+		RunCommandWait("sudo rm -rf /opt")
```
Comment (anupbarve) [inline cmd/decomissionNode.go:119]: I don't think we should be doing this.
This is removal of generic directory that might be used by the customer for other add-ons.

Comment (nikhilbandi) [reply cmd/decomissionNode.go]: @JasminAnt , can you please confirm the removal of subfolders under /opt with the PMK team once?

Comment (JasminAnt) [reply cmd/decomissionNode.go]: image

Comment (JasminAnt) [reply cmd/decomissionNode.go]: @nikhilbandi  who can I message from the team about this? I am not in the slack channel so I dont know who is on that team

File: cmd/decomissionNode.go [hunk 2]
```diff
@@ -130,12 +125,21 @@ func decommissionNodeRun(cmd *cobra.Command, args []string) {
...
-		RunCommandWait("sudo pkill -9 `pidof kubelet`")
-		RunCommandWait("sudo pkill -9 `pidof etcd`")
-		RunCommandWait("sudo pkill -9 `pidof kube_proxy`")
-
 	}
 
+	RunCommandWait("sudo pkill kubelet")
+	RunCommandWait("sudo pkill etcd")
+	RunCommandWait("sudo pkill kube-proxy")
+	RunCommandWait("sudo pkill kube-apiserve")
```
Comment (nikhilbandi) [inline cmd/decomissionNode.go:133]: Isn't it kube-apiserver?

**Review by nikhilbandi — APPROVED** [review]
Approving.

File: cmd/decomissionNode.go (hunk 2 above)
Comment (JasminAnt) [reply cmd/decomissionNode.go:133]: image
Am getting apiserve from netstat

**Pull request merged**